partitioner/src/main/java/io/confluent/connect/storage/partitioner/PartitionerConfig.java

```diff
@@ -101,6 +101,20 @@
   public static final String TIMESTAMP_FIELD_NAME_DEFAULT = "timestamp";
   public static final String TIMESTAMP_FIELD_NAME_DISPLAY = "Record Field for Timestamp Extractor";
 
+  public static final String TIMESTAMP_SCALING_FACTOR_CONFIG = "timestamp.scaling.factor";
```

Comment:
I suggest you break this out as a separate PR

Reply:
@OneCricketeer Sure, I will break into separate PR.